lib/generators/dls_template/templates/.circleci/config.yml

```diff
@@ -0,0 +1,105 @@
+version: 2.1
+orbs:
+  browser-tools: circleci/browser-tools@1.2.5
+  node: circleci/node@5.0.2
+
+commands:
+  install_dependencies:
+    steps:
+      - run: gem install bundler -v '2.2.14'
+      - run: cp Gemfile.lock Gemfile.lock.bak
+      - restore_cache:
+          keys:
+            - dls_template-{{ checksum "Gemfile.lock.bak" }}
```

Comment:
issue: Might be good to try an insert the actual rails application name instead of `dls_template`

Reply:
Handled with gsub in https://github.com/pulibrary/dls_template/pull/6/files#diff-18323fecf0bed6599810f95d774ae826c54df7fadfb2d15a0edb9b1455d82dbfR26